frontend/src/component/feature/FeatureStrategy/FeatureStrategyMenu/FeatureReleasePlanCard/FeatureReleasePlanCard.tsx

```diff
@@ -35,8 +52,13 @@ const StyledCard = styled(Button)(({ theme }) => ({
     borderRadius: theme.spacing(1),
     textAlign: 'left',
     overflow: 'hidden',
-    '&:hover, &:focus': {
-        borderColor: theme.palette.primary.main,
+    // position: 'relative',
+    [`&:hover ${CardContent}`]: {
```

Comment:
Card should gray out.

Reply:
woah. I did not know you could use component names in css selectors like that 👀

Reply:
Also, reducing opacity doesn't really grey it out, it just makes it kinda see-through. It's probably fine here, but if you actually wanna grey it out, you can use [`filter: grayscale(n)`](https://developer.mozilla.org/en-US/docs/Web/CSS/filter) 💁🏼

Reply:
Yeap I think opacity is what we want in this case, as black does not gray out in grayscale. It just stays black, but we need things to go lighter.